cooldowns_monk.lua

```diff
@@ -305,7 +305,6 @@ LCT_SpellData[122470] = {
 	duration = 10,
 	cooldown = 90
 }
--- Mond/Windwalker/talents

```

Comment:
I meant to move the spell above, not remove the header. Ah well, doesn't really matter